monitoring/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
why not move this into the grafana.yaml file?

Reply:
+1

Reply:
namespace.yaml merged into grafana.yaml